tests/test_vrf.py

```diff
@@ -876,7 +876,7 @@ def setup_bgp_with_loopback(self, duthost, ptfhost, cfg_facts):
             ptfhost.shell("ip netns exec {} ip address del {} dev e{}mv1".format(ns, ptf_speaker_ip, vlan_peer_port))
 
         # FIXME workround to overcome the bgp socket issue
-        duthost.shell("vtysh -c 'config terminal' -c 'no router bgp 65444'")
+        #duthost.shell("vtysh -c 'config terminal' -c 'no router bgp 65444'")
```

Comment:
Do you intend to have this change?

Reply:
yes, if the workaround is no longer need, why not remove it.

Reply:
@lguohan
Do you mean that remove line 878 and 879?

Reply:
@prsunny 
May be FRR will update to remove this restriction in the future, so I think that we need to keep it.